pkg/nodeconfig/nodeconfig.go

```diff
@@ -61,6 +62,153 @@ const (
+	// WindowsExporterServiceName is the name of the windows_exporter Windows service
+	WindowsExporterServiceName = "windows_exporter"
+	// AzureCloudNodeManagerServiceName is the name of the azure cloud node manager service
+	AzureCloudNodeManagerServiceName = "cloud-node-manager"
+	// WindowsExporterServiceCommand specifies metrics for the windows_exporter service to collect
+	// and expose metrics at endpoint with default port :9182 and default URL path /metrics
+	WindowsExporterServiceCommand = windowsExporterPath + " --collectors.enabled " +
+		"cpu,cs,logical_disk,net,os,service,system,textfile,container,memory,cpu_info"
+	// serviceQueryCmd is the Windows command used to query a service
+	serviceQueryCmd = "sc.exe qc "
+	// serviceNotFound is part of the error output returned when a service does not exist. 1060 is an error code
+	// representing ERROR_SERVICE_DOES_NOT_EXIST
+	// referenced: https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--1000-1299-
+	serviceNotFound = "FAILED 1060"
+	// cmdExitNoStatus is part of the error message returned when a command takes too long to report status back to
+	// PowerShell.
+	cmdExitNoStatus = "command exited without exit status or exit signal"
+	// removeHNSCommand is the Windows command used to remove HNS network.
+	removeHNSCommand = "Remove-HnsNetwork"
+	// ManagedTag indicates that the service being described is managed by OpenShift. This ensures that all services
+	// created as part of Node configuration can be searched for by checking their description for this string
+	ManagedTag = "OpenShift managed"
+	// containersFeatureName is the name of the Windows feature that is required to be enabled on the Windows instance.
+	containersFeatureName = "Containers"
+	// wicdKubeconfigPath is the path of the kubeconfig used by WICD
+	wicdKubeconfigPath = K8sDir + "\\wicd-kubeconfig"
+	// TrustedCABundlePath is the location of the trusted CA bundle file
+	TrustedCABundlePath = remoteDir + "\\ca-bundle.crt"
+)
+
+var (
+	// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
+	filesToTransfer map[*payload.FileInfo]string
+	// RequiredServices is a list of Windows services installed by WMCO. WICD owns all services aside from itself.
+	// The order of this slice matters due to service dependencies. If a service depends on another service, the
+	// dependent service should be placed before the service it depends on.
+	RequiredServices = []string{
+		WindowsExporterServiceName,
+		KubeProxyServiceName,
+		HybridOverlayServiceName,
+		KubeletServiceName,
+		WicdServiceName,
+		ContainerdServiceName}
+	// RequiredDirectories is a list of directories to be created by WMCO
+	RequiredDirectories = []string{
+		remoteDir,
+		cniDir,
+		CniConfDir,
+		logDir,
+		KubeletLogDir,
+		csiProxyLogDir,
+		KubeProxyLogDir,
+		wicdLogDir,
+		HybridOverlayLogDir,
+		ContainerdDir,
+		containerdLogDir,
+		podManifestDirectory,
+		K8sDir,
+	}
+)
```

Comment:
Bubbling all this information up into the nodeconfig package is not what I'd recommend here. This is overloading the responsibilities of nodeconfig by encoding all of this Windows instance specific information within the package.

Reply:
Like I said before:

>For a starting point I suggest first experimenting with how you can manipulate the [srcDestPairs](https://github.com/openshift/windows-machine-config-operator/blob/55c20d6c68e4fbc6fc1c9ca62d86645f8b726544/pkg/windows/windows.go#L174) when creating the Windows object.

Rather than removing all of this logic from the package itself, I suggest changing the windows.New() function, and the windows struct itself to reflect what needs to be transferred